crates/next-core/js/src/overlay/internal/components/CodeFrame/CodeFrame.tsx

```diff
@@ -67,7 +64,7 @@ export const CodeFrame: React.FC<CodeFrameProps> = function CodeFrame({
 
   // TODO: make the caret absolute
   return (
-    <div data-nextjs-codeframe>
```

Comment:
Some of these identifiers are used in the next.js test suite to find and validate the content of the elements.

Once this is shared with next.js we might need to change the test cases

Reply:
ah ok, good to know